lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -328,8 +328,9 @@ class ReportUIFeatures {
    * @param {ClipboardEvent} e
    */
   onCopy(e) {
-    // Only handle copy button presses (e.g. ignore the user copying page text).
-    if (this._copyAttempt) {
+    // Only handle copy button presses (e.g. ignore the user copying page text) and
+    // when the clipboard is available.
+    if (this._copyAttempt && e.clipboardData) {
```

Comment:
I've seen this error for a long time in VS Code. Did only newer TS emit this error? maybe my editor is set to `typescript@next` ...

Reply:
> I've seen this error for a long time in VS Code. Did only newer TS emit this error? maybe my editor is set to typescript@next ...

they change the browser definitions from time to time, so it's possible. If you do the "Select typescript version" command in vs code do you have "Use VS Code's version" selected?